### DISCUSSION TITLE
UPLOADIFY Downloaded Data Not Working Properly

### DISCUSSION THREAD

**Discussion by juricta** [community/community#112188] [New to GitHub]
I downloaded UPLOADIFY, extracted it and then uploaded the SAMPLE directory files. 
I select 3 pictures (or 2 or whatever) and when I run the UPLOADIFY, it uploads 3 of each photo i choose.  Can someone tell me where I can go to see what I did wrong?

Sorry if I am in the wrong location!

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
